jdaviz/configs/default/plugins/virtual_observatory/vo_plugin.vue

```diff
@@ -0,0 +1,168 @@
+          <v-btn
+            id="autocenterbtn"
+            @click="center_on_data"
+            :disabled="viewer_centered"
+            icon>
+            <v-icon>
+              {{ viewer_centered ? 'mdi-crosshairs-gps' : 'mdi-crosshairs' }}
+            </v-icon>
+          </v-btn>
+        </div>
+      </v-row>
+
+      <v-row>
+        <v-select
+          v-model="coordframe_selected"
+          :menu-props="{ left: true }"
+          attach
+          :items="coordframes"
+          label="Coordinate Frame"
+          hint="Astronomical Coordinate Frame of the provided Coordinates"
+          :disabled="viewer_selected !== 'Manual'"
+          persistent-hint
+        ></v-select>
+      </v-row>
+
+    <j-plugin-section-header>Common Options</j-plugin-section-header>
+
+      <v-row justify="space-between">
+        <div :style="{ width: '55%' }">
+          <v-text-field
+            v-model.number="radius_val"
+            type="number"
+            label="Radius"
+            hint="Angular radius around source coordinates, within which to query for data (Default 1 degree)"
+            persistent-hint>
+          </v-text-field>
+        </div>
+        <div :style="{ width: '40%' }">
+          <v-select
+          v-model="radius_unit_selected"
+          attach
+          :items="radius_unit_items.map(i => i.label)"
+          label="Unit">
+          </v-select>
+        </div>
+      </v-row>
+
+
+    <j-plugin-section-header>Survey Collections</j-plugin-section-header>
+      <v-row>
+        <j-tooltip tipid='plugin-vo-filter-coverage'>
+        <span>
+          <v-btn icon @click.stop="resource_filter_coverage = !resource_filter_coverage">
+            <v-icon>mdi-filter{{ resource_filter_coverage ? '' : '-remove' }}</v-icon>
+          </v-btn>
+          Filter by Coverage
+        </span>
+        </j-tooltip>
+      </v-row>
+      
+      <v-row>
+        <v-select
+          v-model="waveband_selected"
+          :menu-props="{ left: true }"
+          attach
+          :items="wavebands"
+          label="Resource Waveband"
+          hint="Select a spectral waveband to filter your surveys"
+          persistent-hint
+        ></v-select>
+      </v-row>
+
+      <v-row>
+        <div :style="'width: calc(100% - 32px)'">
+          <v-select
+            v-model="resource_selected"
+            :menu-props="{ left: true }"
+            attach
+            :items="resources"
+            :loading="resources_loading"
+            :rules="[() => !!resource_selected || 'This field is required']"
+            label="Available Resources"
+            hint="Select a SIA resource to query"
+            persistent-hint
+          ></v-select>
+        </div>
+        <div style="line-height: 64px; width:32px">
+          <v-btn
+            id="querybtn"
+            @click="query_registry_resources"
+            icon
+            :loading="resources_loading">
+            <v-icon>mdi-refresh</v-icon>
+          </v-btn>
```

Comment:
this button could use a tooltip.  Is there anytime its useful to trigger this manually since it seems to automatically update when changing waveband?

Reply:
Yes! The main reason why that's there is if you edit the form "out of order." There are two main scenarios I've run into where that button was necessary.

1. So imagine you select the waveband first; it will trigger the search, but will return none since you haven't entered a coordinate yet. Then you fill out the source information and return to the resource field, which is still bank. Then you can manually refresh the list yourself.
2. For this scenario, imagine you fill out the form as usual and get your results. But then you either want to go back and select a smaller radius or a smaller source when you filter with coverage. The resource list doesn't refresh because you haven't changed your waveband, so you need that button to force a search.

The auto trigger based on waveband is meant more as a convenience, but can't really be relied on due to the above cases.

I added a tooltip in https://github.com/spacetelescope/jdaviz/pull/2872/commits/ecd4e1cf26542c1effec00a773a91579d3d11ade!